styleguide/components/styleguide.scss

```diff
@@ -93,26 +93,39 @@
   }
 }
 
+.styleguide-table-of-contents {
```

Comment:
I'd remove this whole section.
Don't really see why we have 'styleguide styling'.

Reply:
Because it's styling specifically just to present the elements in Fractal. These specific things never go to the final compiled main.scss

Reply:
OK, out of scope